src/connectivity/mod.rs

```diff
@@ -921,6 +921,32 @@ pub fn articulation_points(graph: &graph::PyGraph) -> HashSet<usize> {
         .collect()
 }
 
+/// Return the bridges of an undirected graph.
+///
+/// A bridge is any edge whose removal increases the number of connected
+/// components of a graph.
+///
+/// .. note::
+///
+///     The function implicitly assumes that there are no parallel edges
+///     or self loops. It may produce incorrect/unexpected results if the
+///     input graph has self loops or parallel edges.
+///
+/// :param PyGraph: The undirected graph to be used.
```

Comment:
There is an indentation error probably around here, that is why the docs job is failing. You can reproduce the error locally with `tox -edocs`